src/components/DataTable/table.scss

```diff
@@ -3,116 +3,128 @@
+	padding-left: 1rem;
 }
 
 thead .data-table-tr th {
-    border-top: none;
+	border-top: none;
 }
 
 .data-table-tbody {
 
-    background-color: $bg-color-dark;
-    & td, th {
-        font-weight: 400;
-        vertical-align: middle;
-    }
+	background-color: $bg-color-dark;
+	& td, th {
+		font-weight: 400;
+		vertical-align: middle;
+	}
 
-    & a {
-        color: $primary;
-    }
+	& a {
+		color: $primary;
+	}
 
-    & .data-table-tr {
-        height: 48px;
-        & .data-table-arrow-btn {
-            padding-left: .75rem;
-            [class^="cil-"], [class*=" cil-"] {
-                color: $primary;
-                cursor: pointer;
-            }
-        }
-    }
+	& .data-table-tr {
+		height: 48px;
+		& .data-table-arrow-btn {
+			padding-left: .75rem;
+			[class^="cil-"], [class*=" cil-"] {
+				color: $primary;
+				cursor: pointer;
+			}
+		}
+	}
 
-    & .data-table-tr-cursor {
-        cursor: pointer;
-    }
+	& .data-table-tr-cursor {
+		cursor: pointer;
+	}
 }
 
 table ~ .react-json-view {
-    border: 1px solid blue;
-    background-color: $bg-color-dark;
+	border: 1px solid blue;
+	background-color: $bg-color-dark;
 }
 
 // STYLES FOR index.js
 
 .data-table {
-    &-input-group-text {
-        padding: 0;
-        background-color: $bg-color;
-        box-shadow: none;
-        border-radius: 2px;
-    }
-    &-search {
-        font-size: 14px;
-        & .form-control {
-            border-left: none;
-            box-shadow: none;
-        }
-    }
-    &-button:last-child {
-        border-radius: 0 8px 0 0;
-    }
-    &-limit {
-        display: flex;
-        align-items: center;
-        // margin: auto 1.5rem;
-        border-right: 1px solid $border-color;
-        &-dropdown {
-            display: flex;
-            justify-content: center;
-            color: $text-color !important;
-            font-weight: 400;
-            margin:  auto;
-            & button {
-                display: flex;
-                justify-content: center;
-                background-color: $bg-color;
-                color: $text-color;
-                font-weight: 400;
-            }
-        }
-    }
-    &-count {
-        justify-self: flex-start;
-    }
+	&-input-group-text {
+		padding: 0;
+		background-color: $bg-color;
+		box-shadow: none;
+		border-radius: 2px;
+	}
+	&-search {
+		font-size: 14px;
+		& .form-control {
+			border-left: none;
+			box-shadow: none;
+		}
+	}
+	&-button:last-child {
+		border-radius: 0 8px 0 0;
+	}
+	&-limit {
+		display: flex;
+		align-items: center;
+		// margin: auto 1.5rem;
+		border-right: 1px solid $border-color;
+		&-dropdown {
+			display: flex;
+			justify-content: center;
+			color: $text-color !important;
+			font-weight: 400;
+			margin:  auto;
+			& button {
+				display: flex;
+				justify-content: center;
+				background-color: $bg-color;
+				color: $text-color;
+				font-weight: 400;
+			}
+		}
+	}
+	&-count {
+		justify-self: flex-start;
+	}
+	&-checkbox {
```

Comment:
sorry about this file. but it was indented using spaces. New lines are ln 110 - ln 121.